DIP3E/Codes/Python/Chapter-3/Track/Gonzalez_example_3_1.py

```diff
@@ -0,0 +1,45 @@
+
+# coding: utf-8
+
+# In[4]:
+
+
+# Gozalez examples 3.1 
+# contrast enhance using gamma transformation 
```

Comment:
It's okay to write the example information as comments. But I recommend that you write it in Markdown cell( see https://github.com/johnnychen94/Digital-Image-Processing-Gonzalez/blob/master/DIP3E/Codes/Julia/Chapter-2/Example2-2.ipynb as an example)

If you do like this now, then you should keep your style consistent for all future work.